### ISSUE TITLE
Library add request: sphinx-multiversion

### ISSUE THREAD

**Issue by BurningEnlightenment** [sphinx-contrib/github-administration#60]
Hello everyone,

the original creator of `sphinx-multiversion` no longer has time to maintain his extension and doesn't oppose appointing other maintainers. We thought it makes sense to move this extension with 149 stars to the `sphinx-contrib` organization to signify the change in maintainership. See Holzhaus/sphinx-multiversion#88

`sphinx-multiversion` is an extension to build and cross-reference multiple versions of a documentation for self-hosting, e.g. on GitHub pages. See also [docs](https://holzhaus.github.io/sphinx-multiversion/master/quickstart.html)

So please invite @Holzhaus, @d33bs and me.

**Comment by stephenfin** [comment]
Hi @BurningEnlightenment,

I have sent you all an invitation to join the GitHub org. After you accept that, you should be able to move the repo yourself (please review the naming convention suggestions at the end of the readme https://github.com/sphinx-contrib/github-administration).

After the move, you may need me to re-grant you admin permission on the repo. If that's the case, comment here on this ticket and let me know.

When we're all done, and you have the permissions you need, we can close this ticket.

Welcome, and thank you for contributing!

**Comment by Holzhaus** [comment]
Thanks, I just accepted the invitation. I'll move the repo later this week.

**Comment by BurningEnlightenment** [comment]
What is the usual way to handle PyPI package ownership for `sphinx-contrib`? Is there a [`spinx-contrib` PyPI organization](https://docs.pypi.org/organization-accounts/)? I guess my first ToDos are to migrate the Travis CI to GitHub Actions and to implement a publishing workflow based on [Trusted Publishing](https://docs.pypi.org/trusted-publishers/) as this will involve Jan Holthuis one way or the other.

**Comment by stephenfin** [comment]
> What is the usual way to handle PyPI package ownership for `sphinx-contrib`? Is there a [`spinx-contrib` PyPI organization](https://docs.pypi.org/organization-accounts/)? I guess my first ToDos are to migrate the Travis CI to GitHub Actions and to implement a publishing workflow based on [Trusted Publishing](https://docs.pypi.org/trusted-publishers/) as this will involve Jan Holthuis one way or the other.

Pretty much that yes. The org currently exists more as a way to publicize projects than share effort maintaining them (like github.com/jazzband is for). That may change organically in the future but it hasn't yet

**Comment by Holzhaus** [comment]
Alright, I moved the repo. I already started some modernization work (CI, build system, etc.) but I'm lacking privileges to configure branch protection, deploy environments, etc. on the repository.

**Comment by dhellmann** [comment]
@Holzhaus , I see you have the maintainer role on the repo, that should let you do everything you need. Maybe @stephenfin set that up since you posted here last, could you try again and see if it's working for you now?

**Comment by Holzhaus** [comment]
@dhellmann I suppose these privileges don't allow changing some repo settings that I need. I only have these settings available:

![image](https://github.com/user-attachments/assets/af60d0cc-2066-4a4d-b403-b96f33d0c732)

Specifically, what I cannot configure is:

- Branch Protection
- Deploy Environments (for publishing to pypi)
- Renaming default branch to`main` ("You don't have permission to rename this branch")
- Enable https://pre-commit.ci for the repo (the repo cannot be selected during setup, although I see some other repos)

I also created a [sphinx-multiversion-maintainers](https://github.com/orgs/sphinx-contrib/teams/sphinx-multiversion-maintainers), but I cannot add the repository to the team (as this apparently requires admin privileges). The team still makes sense for traceability (so in case you ask yourself "why is this dude part of the org" you can use the team membership to identify me as a maintainer for this repo), but will not grant any special privileges.

**Comment by stephenfin** [comment]
> @Holzhaus , I see you have the maintainer role on the repo, that should let you do everything you need. Maybe @stephenfin set that up since you posted here last, could you try again and see if it's working for you now?

Yeah, I meant to set the `admin` role, but GitHub web UI on mobile doesn't scroll properly :sweat_smile: This is done now. Please let us know if you still have issues, otherwise feel free to close this issue.

**Comment by BurningEnlightenment** [comment]
My last open question is: Do we want to add the `sphinx-contrib` PyPI organization as a (backup) co-owner to the PyPI package in order to increase the [bus factor](https://en.wikipedia.org/wiki/Bus_factor)?

@Holzhaus can we briefly chat to clear up the expectations, responsibilities and boundaries going forward? I'd be fine with https://gitter.im, keybase or old-school email.

**Comment by Holzhaus** [comment]
> My last open question is: Do we want to add the `sphinx-contrib` PyPI organization as a (backup) co-owner to the PyPI package in order to increase the [bus factor](https://en.wikipedia.org/wiki/Bus_factor)?

Sure, but I'd like to have the confirmation of one of the maintainers of the `sphinx-contrib` GitHub org that the `sphinx-contrib` user/org on PyPI is actually owned by them (and not some random person who squatted the name) before adding it as co-admin for the package.

**Comment by stephenfin** [comment]
It is, yes. We put in a request to create an equivalent organization on PyPi, which I think has been completed, but as of yet we haven't migrated everything over.

**Comment by Holzhaus** [comment]
is it that one: https://pypi.org/user/sphinx-contrib/ ? Kind of weird profile pic for an org tbh :) Also, I don't find a `sphinx-contrib` user on `test.pypi.org`. Might make sense to have that as well, so that you can manage deployment test projects as well.

**Comment by stephenfin** [comment]
> is it that one: https://pypi.org/user/sphinx-contrib/ ? Kind of weird profile pic for an org tbh :) Also, I don't find a `sphinx-contrib` user on `test.pypi.org`. Might make sense to have that as well, so that you can manage deployment test projects as well.

That would be gravatar, unfortunately 😅 It's linked to my email address (absent another generic one to use).

I'll create the `test.pypi.org` user shortly.

**Comment by stephenfin** [comment]
> Organizations live in a different namespace than users. The one you found is probably from the days before organizations were introduced (like last year, I think). The correct one should be https://pypi.org/org/sphinx-contrib/ (@stephenfin can you confirm this?)

Indeed, we own that also. However, the process for adding to an organization is different and we don't have it in both. I would suggest adding to both for now, if you'd like to close this out today, and I'll update the docs and existing projects next week when I've a chance.

**Comment by Holzhaus** [comment]
I need to check later, no idea how to add it to the org. When I manage owners, I can only add other users, not orgs. Maybe because I'm not a member?

**Comment by stephenfin** [comment]
> I need to check later, no idea how to add it to the org. When I manage owners, I can only add other users, not orgs. Maybe because I'm not a member?

I've sent you an invite now.

**Comment by Holzhaus** [comment]
Thanks, accepted it. Unfortunately it still doesn't work. I found this:

![Screenshot_20241006_015541_com.duckduckgo.mobile.android.png](https://github.com/user-attachments/assets/414955ab-9393-48c3-bc46-b45714c68c9b)

So I guess there is room for improvement for pypi. It should be possible to request transfer without being owner an org after the org owner accepted.

For now, I invited the sphinx-contrib user as co-owner.
